CMakeLists.txt

```diff
@@ -17,27 +17,35 @@ include(cmake/macros.cmake)
-option(FEATURE_LIB_GIT2					"Build LibGit2 support"								 ON)
-option(FEATURE_LIB_PTHREADW32				"Windows only, link to win32 version of pthread"				OFF)
-option(GENERATE_VMMAKER					"If it generates the VMMaker image"						 ON)
-option(GENERATE_SOURCES					"If it generates the C sources"							 ON)
-option(ALWAYS_INTERACTIVE				"Be interactive by default"							OFF)
-option(BUILD_BUNDLE					"Builds a bundle with all dependencies"						 ON)
-option(FEATURE_COMPILE_GNUISATION			"Use gcc gnu extensions to compile the VM"					 ON)
-option(PHARO_DEPENDENCIES_PREFER_DOWNLOAD_BINARIES	"Prefer downloading dependencies"						OFF)
-option(FEATURE_COMPILE_INLINE_MEMORY_ACCESSORS		"Use inline memory accessors instead of macros"					 ON)
-option(PHARO_VM_IN_WORKER_THREAD			"Have support for pharo running in a different thread that the main one"	 ON)
-option(DEPENDENCIES_FORCE_BUILD         "Force build libraries" OFF)
-option(BUILD_WITH_GRAPHVIZ              "Generate dependency graphs" ON)
-option(VERSION_UPDATE_FROM_GIT              "Extract version information from git tags. Default to true. Follow vX.Y.Z-suffix" TRUE)
+option(VERBOSE_BUILD			"Verbose Build"														OFF)
+option(FEATURE_FFI				"Enable FFI"														ON)
+option(FEATURE_THREADED_FFI		"Enable Threaded (running in another thread) FFI"					ON)
+option(FEATURE_MESSAGE_COUNT	"Enable the option to count messages, only valid for StackVM"		OFF)
+option(FEATURE_NETWORK			"Enable network and sockets"										ON)
+option(FEATURE_JIT_SIMD			"Use SIMD support in JIT compilation when available"				ON)
+option(FEATURE_LIB_SDL2			"Build SDL2 support"												ON)
+option(FEATURE_LIB_CAIRO		"Build Cairo support"												ON)
+option(FEATURE_LIB_FREETYPE2	"Build freetype2 support"											ON)
+option(FEATURE_LIB_GIT2			"Build LibGit2 support"												ON)
+option(FEATURE_LIB_PTHREADW32	"Windows only, link to win32 version of pthread"					OFF)
+option(GENERATE_VMMAKER			"If it generates the VMMaker image"									ON)
+option(GENERATE_SOURCES			"If it generates the C sources"										ON)
+option(ALWAYS_INTERACTIVE		"Be interactive by default"											OFF)
+option(BUILD_BUNDLE				"Builds a bundle with all dependencies"								ON)
+option(FEATURE_COMPILE_GNUISATION
+								"Use gcc gnu extensions to compile the VM"							ON)
+option(FEATURE_COMPILE_INLINE_MEMORY_ACCESSORS
+								"Use inline memory accessors instead of macros"						ON)
+option(PHARO_DEPENDENCIES_PREFER_DOWNLOAD_BINARIES
+								"Prefer downloading dependencies"									OFF)
+option(PHARO_VM_IN_WORKER_THREAD
+								"Run the VM in a thread different that the main"					ON)
+option(DEPENDENCIES_FORCE_BUILD
+								"Force build libraries"												OFF)
+option(BUILD_WITH_GRAPHVIZ
+								"Generate dependency graphs"										ON)
+option(VERSION_UPDATE_FROM_GIT
+								"Extract version information from git tags. Default to true. Follow vX.Y.Z-suffix" ON)
```

Comment:
I formatted a bit this code, the actually important line is line 25 adding

FEATURE_JIT_SIMD